### PR TITLE
Add LIDAR profile to desktop alt

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: contribs/gmf/apps/desktop_alt/Controller.js [hunk 1]
```diff
@@ -36,6 +36,7 @@ import gmfControllersAbstractDesktopController, {
 import appBase from '../appmodule.js';
 import gmfImportModule from 'gmf/import/module.js';
 import gmfFloorModule from 'gmf/floor/module.js';
+import gmfLidarprofileModule from 'gmf/lidarprofile/module.js';
```
Comment (ger-benjamin) [inline contribs/gmf/apps/desktop_alt/Controller.js:39]: Missing`@import '~gmf/lidarprofile/lidarprofile.scss';`

**Review by ger-benjamin — COMMENTED** [review]
Looks good but do we have data ? I've not seen the equivalent of the old: `module.constant('pytreeLidarProfileJsonUrl', 'https://<domain>/pytree/pytree_dev/');` You have one to add in the demo ?

(for me, old integration ref: https://github.com/camptocamp/ngeo/pull/3855)

**Comment by sbrunner** [comment]
> Looks good but do we have data ? I've not seen the equivalent of the old: module.constant('pytreeLidarProfileJsonUrl', 'https://<domain>/pytree/pytree_dev/'); You have one to add in the demo ?

=> https://github.com/camptocamp/demo_geomapfish/commit/ff6d0c36dfffc95c2527ca539dad7c3643aa4d7e

File: contribs/gmf/apps/desktop_alt/Controller.js (hunk 1 above)
Comment (sbrunner) [reply contribs/gmf/apps/desktop_alt/Controller.js:39]: In lidarprofile we already have: `import './lidarprofile.scss';`...

Comment (ger-benjamin) [reply contribs/gmf/apps/desktop_alt/Controller.js:39]: Okay... the style looks weird though...

**Review by ger-benjamin — APPROVED** [review]
@sbrunner as I'll work on GSGMF-1486 I merge this PR.
I'll open a new PR for the ticket and I'll check the styling problem as well.

**Pull request merged**